services/repository/transfer.go

```diff
@@ -177,6 +178,22 @@ func transferOwnership(ctx context.Context, doer *user_model.User, newOwnerName
 		}
 	}
 
+	// Remove project's issues that belong to old organization's projects
+	if oldOwner.IsOrganization() {
+		projects, err := project_model.GetAllProjectsIDsByOwnerIDAndType(ctx, oldOwner.ID, project_model.TypeOrganization)
```

Comment:
So then we need a warning popup window to warn that some project information will be removed but not deleting them silently.

Reply:
![Screenshot 2024-08-06 at 21 02 50](https://github.com/user-attachments/assets/b0f04884-bf4e-4d57-95a4-8bff6724ca06)

maybe I can add one more warning to this already existing warning module.